js/contentAssessor.js

```diff
@@ -90,11 +118,11 @@ ContentAssessor.prototype.calculatePenaltyPoints = function() {
 	var penaltyPoints = map( results, function( result ) {
 		var rating = scoreToRating( result.getScore() );
 
-		if ( this.getPaper().getLocale().indexOf( "en_" ) > -1 ) {
-			return this.calculatePenaltyPointsEnglish( rating );
+		if ( this.isLanguageFullySupported() ) {
```

Comment:
It would be nice to have two objects to do this. One for a supported language and one for a unsupported one. If you do that, there is no reason for this if.